errors/errors.js

```diff
@@ -0,0 +1,16 @@
+exports.handleCustomErrors = (err, req, res, next) => {
+    if (err.status && err.msg) {
+        res.status(err.status).send({ msg: err.msg })
+    } else next(err)
+}
+
```

Comment:
We don't need to include PSQL error handling just yet, please remove this until it's needed 👍